### DISCUSSION TITLE
Locally installed extensions do not work by default with snap-installed Foxglove

### DISCUSSION THREAD

**Discussion by troygibb** [foxglove/community#1000] [Bugs] [answered]
I had previously installed Foxglove using `snap`. When I pulled down a local extension that I built on my machine, it correctly placed the extension in `~/.foxglove-studio/extensions`, but when I reloaded Foxglove, the extension did not show up on my panel list. 

When I re-installed Foxglove via manually downloading the binary, the panel was correctly listed.

OS: Ubuntu 22.04
Foxglove Version: 2.4.0

**Comment by jtbandes** [comment, marked as answer]
What does this mean: 
> When I pulled down a local extension that I built on my machine

This was resolved for `yarn local-install` as follows: https://github.com/foxglove/create-foxglove-extension/pull/91

**Comment by troygibb** [comment]
I've been making changes to https://github.com/yulong88888/foxglove-nipple, I pulled it down and have been running `npm run local-install` which is where I noticed the issue. 

It looks like it's on an old version of `create-foxglove-extension` (`v0.6.1`). I'll update! Thanks!!
